packages/amplify-graphql-model-transformer/src/__tests__/amplify-dynamodb-table-construct.test.ts

```diff
@@ -115,4 +115,20 @@ describe('Amplify DynamoDB Table Construct Tests', () => {
     // Error will be thrown when index does not exist
     expect(() => table.schema('notExist')).toThrow();
   });
+  it('render the correct template when destructive updates are allowed', () => {
+    const stack = new cdk.Stack();
+    new AmplifyDynamoDBTable(stack, 'MockTable', {
+      customResourceServiceToken: 'mockResourceServiceToken',
+      allowDestructiveGraphqlSchemaUpdates: true,
+      tableName: 'mockTableName',
+      partitionKey: {
+        name: 'id',
+        type: AttributeType.STRING,
+      },
+    });
+    const template = Template.fromStack(stack);
+    template.hasResourceProperties(CUSTOM_DDB_CFN_TYPE, {
+      allowDestructiveGraphqlSchemaUpdates: true,
+    });
+  });
 });
```

Comment:
As noted below, let's add a test that verifies destructive operations throw an error if the flag isn't specified.

Reply:
Disregard, saw https://github.com/aws-amplify/amplify-category-api/pull/2039/files#diff-0a311dff043ffa5e6e67f34000f2e8fee12b539efb486f9461c285b1ffe561ceR49